launch/tier4_simulator_launch/launch/simulator.launch.xml

```diff
@@ -45,6 +45,15 @@
           <arg name="use_vector_map" value="true" />
         </include>
       </group>
+
+      <!-- publish empty objects instead of object recognition module -->
+      <group unless="$(var perception/enable_object_recognition)">
```

Comment:
It seems this tag is inside `<group if="$(var perception/enable_object_recognition)">`, which will never be true.

Reply:
I fixed at [b0eabbb](https://github.com/autowarefoundation/autoware.universe/pull/613/commits/b0eabbb753cf59ffafc1fe70d7a154e81e23063f)

Reply:
I also fixed the indent.